lib/concurrent-ruby/concurrent/utility/processor_counter.rb

```diff
@@ -107,4 +145,20 @@ def self.processor_count
   def self.physical_processor_count
     processor_counter.physical_processor_count
   end
+
+  # Number of processors usable for process scheduling.
+  # Returns `nil` if there is no quota, or a `Float` if the
```

Comment:
```suggestion
  # Returns `nil` if there is no #cpu_quota, or a `Float` if the
```
Ideally it'd link to the other method, not sure the correct way to do that with YARD.

Reply:
AH also this doesn't return `nil` so probably worth a while pass over the docs of this method.
It should also mention it uses `processor_count` aka logical processor count.